src/dbus.c

```diff
@@ -1,15 +1,14 @@
 /* copyright 2013 Sascha Kruse and contributors (see LICENSE for licensing information) */
-
-#include <stdlib.h>
-#include <stdio.h>
+#include "dbus.h"
```

Comment:
To ease readability, we keep the local import under the external ones with an empty line separating them. This happens on a lot of other files, I won't comment on all of them for this issue but they should also be fixed.